runtime/tr.source/trj9/codegen/J9CodeGenerator.cpp

```diff
@@ -666,7 +666,10 @@ J9::CodeGenerator::lowerTreesPreChildrenVisit(TR::Node *parent, TR::TreeTop *tre
       if (self()->comp()->useCompressedPointers())
          {
          TR::MethodSymbol *methodSymbol = parent->getSymbol()->castToMethodSymbol();
+         // Unsafe could be the jdk.internal JNI method or the sun.misc ordinary method wrapper,
```

Comment:
Please include that this is the case in Java 9 up - 8 and earlier will have the sun.misc.Unsafe as the native.